tasks/pull.yaml

```diff
@@ -32,3 +32,31 @@ tasks:
             CURRENT_PKG_NAME=$(cat zarf.yaml | yq .metadata.name)
             test -f zarf-package-${CURRENT_PKG_NAME}-${UDS_ARCH}-${CURRENT_VERSION}.tar.zst || mv zarf-package-${CURRENT_PKG_NAME}-${UDS_ARCH}-*.tar.zst zarf-package-${CURRENT_PKG_NAME}-${UDS_ARCH}-${CURRENT_VERSION}.tar.zst
           fi
+
+  - name: latest-bundle-release
+    inputs:
+      path:
+        description: Path relative to the repositories root where the bundle things happen
+        default: .
+      spoof_release:
+        description: Whether to spoof the pulled bundle version to the current repo version
+        default: "false"
+    actions:
+      - description: Get the current uds bundle name
+        cmd: cat ${{ .inputs.path }}/uds-bundle.yaml | yq .metadata.name
+        setVariables:
+          - name: BUNDLE_NAME
+      - description: Get latest tag version from OCI
+        cmd: ./uds zarf tools registry ls ${TARGET_REPO}/bundles/${BUNDLE_NAME} | sort -V | tail -1
+        setVariables:
+          - name: LATEST_VERSION
+      - description: Pull the latest bundle release
+        cmd: ./uds pull oci://${TARGET_REPO}/bundles/${BUNDLE_NAME}:${LATEST_VERSION} --no-progress -o ${{ .inputs.path }}
```

Comment:
TARGET_REPO could probably be an input like publish is now

Reply:
i was keeping parity with the pull-package. fine with updating both